v0/destinations/attribution/transform.js

```diff
@@ -36,10 +36,9 @@ function getTransformedJSON(message) {
   if (traits && traits.anonymousId) {
     delete traits.anonymousId;
   }
-  const properties = get(message, "properties")
-    ? message.properties
-    : undefined;
-  const event = get(message, "event") ? message.event : undefined;
+  const properties = get(message, "properties");
+  const event = get(message, "event");
+  const context = get(message, "context");
```

Comment:
We don't need to use `get` for accessing immediate children. `object.childName` should do.

Reply:
Fixed that too.